package.json

```diff
@@ -68,6 +68,7 @@
     "proxy-polyfill": "0.1.6",
     "re-reselect": "^4.0.0",
     "react-beautiful-dnd": "^13.1.0",
```

Comment:
Tag overflow in search suggestions.  It's minor, since the original (touching the border) is also bad; 

![image](https://user-images.githubusercontent.com/64950861/150543895-ceaa04cc-d26a-45ba-bcf7-51731af8f414.png)

![image](https://user-images.githubusercontent.com/64950861/150543931-3d7c449f-d4c4-4172-b078-0edc5c7133d6.png)